README.md

```diff
@@ -1,6 +1,48 @@
 # Storage-Less HTTP Sessions
 
-**WARNING: THIS REPOSITORY IS A MOCKUP! DO NOT ATTEMPT USING THIS YET!**
+### Installation
+
+```sh
+composer require ocramius/storage-less-session
+```
+
+### Usage
+
+You can use the `StoragelessSession\Http\SessionMiddleware` in any 
+[`zendframework/zend-stratigility`](https://github.com/zendframework/zend-stratigility)
```

Comment:
Mostly stratigility, as it defines the interface. If you want, you can send a PR with examples for Slim3 or RelayPHP :-D